std/assembly/typedarray.ts

```diff
@@ -98,4 +327,27 @@ export class Float64Array extends TypedArray<f64,f64> {
   subarray(begin: i32 = 0, end: i32 = 0x7fffffff): Float64Array {
     return changetype<Float64Array>(super.subarray(begin, end));
   }
+
+  /**
+   * @param callbackfn {function} - a function that reduces each value to a ReturnType
+   * @param initialValue {ReturnType} - the initial ReturnType value to be passed to the callbackfn
+   */
+  reduce<ReturnType>(
+    callbackfn: (accumulator: ReturnType, value: f64, index: i32, array: Float64Array) => ReturnType,
+    initialValue: ReturnType,
+    ): ReturnType {
```

Comment:
Just a minor one, but compared to other code, this should indent by 2 less.

Reply:
I'm sorry, I don't understand?

Reply:
I can format this code anyway that you would like. Can I just have a short example?

Reply:
Just committed. @dcodeIO please review.

Thanks!